config/opal_setup_wrappers.m4

```diff
@@ -220,18 +228,19 @@ EOF
 AC_DEFUN([OPAL_SETUP_RUNPATH],[
     OPAL_VAR_SCOPE_PUSH([LDFLAGS_save rpath_script rpath_outfile wl_fc])
 
-    AC_MSG_CHECKING([if linker supports RUNPATH])
     # Set the output in $runpath_args
     runpath_args=
     LDFLAGS_save=$LDFLAGS
     LDFLAGS="$LDFLAGS -Wl,--enable-new-dtags"
-    AC_LANG_PUSH([C])
-    AC_LINK_IFELSE([AC_LANG_PROGRAM([], [return 7;])],
-                   [WRAPPER_RPATH_SUPPORT=runpath
-                    runpath_args="-Wl,--enable-new-dtags"
-                    AC_MSG_RESULT([yes (-Wl,--enable-new-dtags)])],
-                   [AC_MSG_RESULT([no])])
-    AC_LANG_POP([C])
+    AS_IF([test x"$enable_wrapper_runpath" = x"yes"],
```

Comment:
Ah, because of `AS_IF([test "$enable_wrapper_rpath" != "no"], [enable_wrapper_rpath=yes])`, up above.  Got it.